internal/tiltfile/tiltfile.go

```diff
@@ -186,6 +189,9 @@ func (tfl tiltfileLoader) Load(ctx context.Context, filename string, userConfigS
 	tlr.Manifests = manifests
 	tlr.TeamID = s.teamID
 
+	objectSet, _ := v1alpha1.GetState(result)
+	tlr.ObjectSet = objectSet
```

Comment:
Why do we ignore errors from all these? It's consistent with the others so not suggesting changing now, but would it make sense to create an aggregate error for assignment to `tlr.Error` in the future?

Reply:
ya, I'm going to change them to all use MustState (which panics on error)

The problem is that an error here is a hard-stop programmer error - in particular, that you're trying to get the state of a plugin, but never registered that plugin.

Honestly, my vision for this is that TiltfileLoadResult goes away entirely, and is almost entirely a collection of API objects and/or Plugin state

Reply:
wait, nope, the original code was correct. I added this comment:

```
	// All data models are loaded with GetState. We ignore the error if the state
	// isn't properly loaded. This is necessary for handling partial Tiltfile
	// execution correctly, where some state is correctly assembled but other
	// state is not (and should be assumed empty).
```

Reply:
(it would not be correct to aggregate + report the GetState errors, since that would just obscure the original error)

Reply:
ahhhh ok that makes sense